docs/kyma/docs/029-details-install-subcomponents.md

```diff
@@ -43,4 +43,24 @@ data:
   azure-broker.enabled: "true"
 ```
 
->**NOTE:** Some subcomponents can require additional configuration to work properly.
\ No newline at end of file
+>**NOTE:** Some subcomponents can require additional configuration to work properly.
+
+## Specify subcomponents versions
+
+Versions of the Kyma components are specified in `values.yaml` file in charts. Each component version is described by two properties: `version` and `dir`. The first defines actual docker image tag. The second describes directory under which the tagged image is pushed follwed by a `/` or this value can be empty.
+
+Possible values of `dir` property:
+- `pr/` - contains images built from Pull Request
+- `develop/` - contains images built from master branch
+- `rc/` - contains images built for pre-release
+- `` - (empty) contains images built for release
+
+To override subcomponents versions during Kyma startup, create `versions-overrides.env` file in `installation` directory. The example overrides Environments component setting its image version to `0.0.1` from `develop` directory
```

Comment:
- ``the `versions-overrides.env` file in the `installation` directory.``
- maybe some additional line here to show where the example description starts?
-  ``The example overrides the `Environments` component and sets the image version to `0.0.1`, based on the version from the `develop` directory.``